params/config.go

```diff
@@ -491,6 +493,16 @@ func (o *OptimismConfig) String() string {
 	return "optimism"
 }
 
+// CeloConfig is the celo config.
+type CeloConfig struct {
+	EIP1559BaseFeeFloor uint64 `json:"eip1559BaseFeeFloor"`
```

Comment:
Is it only me, or does putting EIP-1559 into the name make is sound like it only applies to EIP-1559 txs and not legacy txs?

Reply:
I've tried to maintain the same way that optimism is naming the base fee parameters. The eip1559 is about the baseFee, maybe the Tx shouldn't have the 1559 on it, haha.